### DISCUSSION TITLE
越來越開放的 Windows 來了！微軟直接為 Windows 11 提供 Android 子系統

### DISCUSSION THREAD

**Discussion by Rosancl** [1101-GS4719-C/Python-Community-GS4719-C#644] [General]
微軟（Microsoft）在近日宣布了一項新的更新，將升級使用者 PC 上運作的 Android 子系統至 12.1 版本，雖然目前僅在 Windows Insider當中進行測試。

此次的更新包含了把 Android 的[彈出訊息]（浮動式訊息）顯示成 Windows 通知，Windows 上的任務欄也可以顯示 Android 應用程式是否有在存取麥克風或位置。若使用者將 PC 從待機狀態喚醒，Android 應用程式會從上次中斷的狀態開始，而非需要重新啟動。

微軟還指出，改善了網路功能之後，使用者未來可以以 PC 上的 Android App 設定智慧家居設備。

https://buzzorange.com/techorange/2022/05/23/windows-11-android/

等正式推出更新版本以後，Windows和Android連動應該是會方便很多。

**Comment by sys511613** [comment]
希望能早點開放windows和手機之間的鄰近分享
現在鄰近分享貌似只能電腦傳電腦、手機傳手機，或是純粹我不會用@@

**Comment by yucheng1028** [comment]
終於阿 想開始搞微軟生態系了

**Comment by leon10456** [comment]
感謝樓主的分享 現在才開始好像有點太晚 市場飽和了

**Comment by liangyinglly** [comment]
感覺這方面蘋果領先許多

**Comment by stan890425** [comment]
開發下去就不用裝外掛就可以在電腦玩手機了

**Comment by jason79461385** [comment]
之前有看過類似的文章 好像是在電腦上玩傳說對決之類的遊戲

**Comment by bloodtrail0511** [comment]
早就該這麼開放了微軟:(

**Comment by etin2468** [comment]
我是有使用過了，但感覺還是蠻實驗性的，bug還很多

**Comment by Hchejxnc** [comment]
期待未來他們技術的成熟
